docs/configuration.md

```diff
@@ -65,6 +65,7 @@ Property
 -------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------
 **envoy-control.envoy.snapshot.incoming-permissions.enabled**                                                                                      | Enable incoming permissions                                                                                                                                                                                      | false
 **envoy-control.envoy.snapshot.incoming-permissions.client-identity-headers**                                                                      | Headers that identify the client that called the endpoint. In most cases `client-identity-header` should include `service-name-header` value to correctly identify other services in the mesh.                   | [ x-service-name ]
+**envoy-control.envoy.snapshot.incoming-permissions.request-identification-headers**                                                               | Headers that are used to identify requests in incoming permissions logs.                                                                                                                                         | [ x-request-id ]
```

Comment:
How about generalizing `client-identity-headers` and `request-identification-headers`. We could provide a configuration for a list of interesting headers and just log them all rather than first matching from these two mentioned lists.

Reply:
> log them all 

I do not think we want to log them all. It would be more of a list of lists and the first position that matches in the list is used. So it's a question of do we want two list or a list of lists?